frontend-opal/src/components/investment.js

```diff
@@ -1,14 +1,23 @@
+import React from 'react';
 import { createAppContainer } from 'react-navigation';
 import { createStackNavigator } from 'react-navigation-stack';
 
 import MainEntry from './MainEntry';
-import Portfolio from './PortfolioContainer';
 import Search from './SearchResult/Result';
+import Portfolio from './PortfolioContainer';
+import Watchlist from './WatchList';
+import AddTabs from './common/AddTabs';
+import headers from './common/headers';
+
+const InvestmentMain = () => {
+  return <AddTabs Cmpnts={[Portfolio, Watchlist]} names={['PORTFOLIO', 'WATCHLIST']} />;
```

Comment:
@hondahmd instead of putting the title into a separate array, you can attach them to the components:
```javascript
Portfolio.title = 'PORTFOLIO'
// or
Portfolio.tabTitle = 'PORTFOLIO'
```